cdb2api/cdb2api.c

```diff
@@ -4763,7 +4763,7 @@ int cdb2_open(cdb2_hndl_tp **handle, const char *dbname, const char *type,
         char *p;
         p = strchr(type, ':');
         if (p) {
-            strcpy(hndl->hosts[0], type);
+            strncpy(hndl->hosts[0], type, sizeof(hndl->hosts[0]));
```

Comment:
`hndl` is allocated via `calloc` so it is fine to leave `NUL`. However I'd agree that explicit nul termination definitely improves the readability.